### ISSUE TITLE
Show webhook requests/deliveries in the store admin UI

### ISSUE THREAD

**Issue by woutersamaey** [btcpayserver/btcpayserver#2549]
The webhook requests/deliveries + feature to resend webhook deliveries should be in the admin UI.
These are available through the Greenfield API, but can't seem to find them in the backoffice.

Would be helpful to improve debugging.

**Comment by Kukks** [comment]
They are in the invoice

**Comment by woutersamaey** [comment]
Nice!

Still would wanna keep this open for a broader overview and maybe a button "resend all" ?

It may be tedious if you need to resend a lot of them.

Putting it in the store view may also not be ideal. Perhaps it should be system-wide? That's up for debate.

**Comment by Kukks** [comment]
Yeah, we'll need to think about it more as later on there will be webhooks
not relating to an invoice and webhooks NOT related to a store even.

**Comment by woutersamaey** [comment]
I was also thinking that any "resend all" may be a bad idea, since some webhooks are updates of previous ones, so ideally:
- You should only send the latest webhook event for a given object
- The receiving party should NOT use any of the data it receives, only the ID and fetch the latest status from the server.

This last point also means that a webhook should not contain so much information as it does now. Only the event name and the ID of what it's about. I know that with Greenfield, we now have a shared `secret` that allows the receiver to trust the sender, but while that is nice, it does encourage bad practice of using the (possibly stale) received data instead of fetching fresh data.

Another option is that you block old webhook requests from (re-)firing if there is a newer one and this should have sub-second accuracy.

So yeah, lots to talk about.

**Comment by pavlenex** [comment]
@Kukks @woutersamaey what do we want to do with this one? Close, work on it?

**Comment by woutersamaey** [comment]
I think we should ultimately do it, but let me first look into the webhook structure as it is. First I wanna add more webhooks and this could be a follow up to that.

**Comment by pavlenex** [comment]
Moving to discussions due to lack of interest, will repopen if enough reuquests from community.
